go/src/socialapi/workers/payment/tests.sh

```diff
@@ -3,11 +3,6 @@
 echo "running payment tests"
 go test -c socialapi/workers/payment/ && ./payment.test -c $(dirname $0)/../../config/dev.toml -test.v=true
 
-echo "running stripe tests"
-go test -c socialapi/workers/payment/stripe/ && ./stripe.test -c $(dirname $0)/../../config/dev.toml -test.v=true
+echo "running api tests"
+go test -c socialapi/workers/payment/api && ./api.test -c $(dirname $0)/../../config/dev.toml -test.v=true
```

Comment:
Interesting suggestion, i will experiment with it to make it compatible with other workers/tests - currently "runner" does the flag registration/parsing part